aten/src/ATen/core/jit_type.h

```diff
@@ -649,6 +649,7 @@ struct CAFFE2_API DictType : public Type {
       case TypeKind::FloatType:
       case TypeKind::StringType:
       case TypeKind::TensorType:
+      case TypeKind::VarType:
```

Comment:
How can a `VarType` that is not one of the specified key types work as a dict key?

Reply:
Otherwise it fails in schema parsing, since the dict is first instantiated with vartype